bot/exts/utils/package.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import random
+from typing import Dict, Union
+
+from aiohttp.client_exceptions import ContentTypeError
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+log = logging.getLogger(__name__)
+
+pypi_api_url = "https://pypi.org/pypi/{package_name}/json"
+
+
+class Package(commands.Cog):
+    """Provides Information about package hosted at pypi."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @staticmethod
+    def embed_builder(pypi_pkg: dict) -> Embed:
+        """Returns informative embed about package."""
+        embed = Embed()
+        embed.colour = Colours.soft_orange
+        embed.set_thumbnail(url="https://miro.medium.com/max/660/1*2FrV8q6rPdz6w2ShV6y7bw.png")
+
+        links = []
+
+        for pkg_info_heading, pkg_info in pypi_pkg.items():
+
```

Comment:
I don't think that this newline is necessary.

Reply:
I'm not also sure about generation, this just don't feel good way to do this.

Reply:
actually i am trying to make embed field dynamic thats y i have done that bcz some package have some field and some isnt